internal/sessiontest/session_test.go

```diff
@@ -28,7 +29,15 @@ func TestSigningSession(t *testing.T) {
 func TestDisclosureSession(t *testing.T) {
 	id := irma.NewAttributeTypeIdentifier("irma-demo.RU.studentCard.studentID")
 	request := getDisclosureRequest(id)
-	sessionHelper(t, request, "verification", nil)
+	responseString := sessionHelper(t, request, "verification", nil)
+
+	// Check default JWT validity
+	claims := jwt.MapClaims{}
+	jwt.ParseWithClaims(responseString, claims, nil)
```

Comment:
This ignores the error parameter that this function can return; you should catch it and feed it to `require.NoError`.

Reply:
Catching the error is nicer indeed. It was ignored as I first only checked for the difference in expiration time and issueing time. In a new commit, a function with the corresponding public key is passed to the ParseWithClaims and the error is taken care of.